recipes/r-txtq/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-txtq
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/txtq_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/txtq/txtq_{{ version }}.tar.gz
+  sha256: 3d29b4460667a8be312cd3da2dd870c6aa7bd1f4b5d46ef1e6e06875e6f87d63
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-r6
+    - r-base64url
+    - r-filelock
+  run:
+    - r-base
+    - r-r6
+    - r-base64url
+    - r-filelock
+
+test:
+  commands:
+    - $R -e "library('txtq')"           # [not win]
+    - "\"%R%\" -e \"library('txtq')\""  # [win]
+
+about:
+  home: https://github.com/wlandau/txtq
+  license: MIT
+  summary: This queue is a data structure that lets parallel processes send and receive messages,
+    and it can help coordinate the work of complicated parallel tasks. Processes can
+    push new messages to the queue, pop old messages, and obtain a log of all the messages
+    ever pushed. File locking preserves the integrity of the data even when multiple
+    processes access the queue simultaneously.
+  license_family: MIT
+  license_file: LICENSE.md
```

Comment:
Please change this to:
```
license_file:
  - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
  - LICENSE
``` 

If you update your installed conda-build version, this should have been done this way already.

Reply:
Sorry, didn't see this before making the other change.  Thanks for the help!